docs/recipes/Deploying-Modules.md

```diff
@@ -0,0 +1,55 @@
+[👈 Return to Overview](./Recipes.md)
+
+# Deploying Modules
+
+## 📖 Table of Contents
+
+* [Deploying to Now with GitHub Actions](#deploying-to-now-with-github-actions)
```

Comment:
is a TOC needed with only 1 item? or this is for consistency ... if its for consistency then ok

Reply:
I have it for consistency for when we eventually add more ways.